resources/scenic-analysis.wrattler

```diff
@@ -0,0 +1,157 @@
+# Testing the Broken Window hypothesis: are less scenic neighbourhoods linked to more crime?
+
+## Crime data
+
+The first dataset we import is the [MPS LSOA Level Crime (historic)](https://data.london.gov.uk/dataset/recorded_crime_summary) dataset.
+This contains counts of the number of crimes at different Lower Super Output Area (LSOA) geographic locations in London per month, according to crime type. 
+
```

Comment:
Let's add: "LSOAs are geographic areas with an average population size of 1,600, defined by the Office of National Statistics for statistical analyses, with areas ranging between 0.018 square km to 684 square km."